torch/_inductor/overrides.py

```diff
@@ -587,7 +645,46 @@ def fuse_conv_bn(gm: torch.fx.GraphModule, inplace=False):
                 node.replace_all_uses_with(node.args[0])
                 gm.graph.erase_node(node)
                 gm.graph.lint()
+    for pattern in module_function_patterns:
+        for node in gm.graph.nodes:
+            if matches_module_function_pattern(pattern, node, modules):
+                if len(node.args[0].users) > 1:  # Output of conv is used by other nodes
+                    continue
+                if len(node.args) != 8:
```

Comment:
If we prefer to only handle args, not kwargs here, please add a note (or maybe TODO) on the limitation.

Reply:
Yes, comment is added.